event.go

```diff
@@ -148,14 +148,14 @@ func UnmarshalEvents(anns Annotations, events *[]Event) error {
 	return nil
 }
 
-// A spanName event sets a span's name.
-type spanName struct{ Name string }
+// A SpanNameEvent event sets a span's name.
+type SpanNameEvent struct{ Name string }
```

Comment:
```
rename+expose s/spanName/SpanEventEvent/g
expose s/schemaPrefix/SchemaPrefix/g
```
not really clear to me why this is necessary when just viewing the commit and commit message. Although my guess is it has something to do with moving the influx store (haven't searched looked at all the commits just yet)?